### PR TITLE
add polar file load to repl example

### PR DESCRIPTION
**Pull request by saolsen**

### CODE REVIEW

File: docs/content/node/reference/tooling/repl/data/data.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ replApi: |
     const oso = new Oso();
     oso.registerClass(Expense);
     oso.registerClass(User);
+    oso.loadFiles(["alice.polar"])
```
Comment (dhatch) [inline docs/content/node/reference/tooling/repl/data/data.md:50]: Needs await, right?

**Review by dhatch — CHANGES_REQUESTED** [review]

File: docs/content/node/reference/tooling/repl/data/data.md (hunk 1 above)
Comment (saolsen) [reply docs/content/node/reference/tooling/repl/data/data.md:50]: uhhh yeah I think so

Comment (saolsen) [inline docs/content/node/reference/tooling/repl/data/data.md:50]:
```suggestion
    await oso.loadFiles(["alice.polar"])
```

Comment (dhatch) [reply docs/content/node/reference/tooling/repl/data/data.md:50]: lgtm after that

**Review by dhatch — APPROVED** [review]

**Pull request merged**